cases/models.py

```diff
@@ -107,7 +107,7 @@ def get_completed_calls(self):
         ).exclude(status="Planned")
 
     def get_assigned_user(self):
-        return User.objects.get(id=self.assigned_to.id)
+        return User.objects.get(id=self.assigned_to_id)
```

Comment:
Django caches foreignkey ids, so doing this means one less db read. [read more](https://django.doctor/advice/C9001)